frontend/components/page-contents/feedback-task-content/CenterFeedbackTask.content.vue

```diff
@@ -50,6 +57,17 @@ export default {
     totalRecords() {
       return getTotalRecordByDatasetId(this.datasetId);
     },
+    recordStatusFilteringValue() {
+      const { _status: recordStatus } = this.$route.query;
+      return recordStatus ?? RECORD_STATUS.PENDING.toLowerCase();
+    },
+  },
+  watch: {
+    async recordStatusFilteringValue() {
+      // NOTE - each time the filter change, clean records orm && rerender the children component
+      await deleteAllRecords();
```

Comment:
Regarding the ORM refresh when the response status filter changes, we need to adapt the change introduced in https://github.com/argilla-io/argilla/pull/2851. We should prevent users if some data may get lost. 

If this change is big, you can tackle it in another PR. In that case, please, note this somewhere before merging the branch

Reply:
Ref : #2863